lib/components/toggle.dart

```diff
@@ -36,25 +34,19 @@ class _ToggleState extends State<Toggle> {
                 }),
                 side: MaterialStateProperty.all(
                   BorderSide(
-                    color: $styles.colors.black,
-                    width: 2,
+                    color: $styles.borderStyle.stroke,
+                    width: $styles.borderStyle.width,
                   ),
                 ),
               ),
               segments: <ButtonSegment<String>>[
                 ButtonSegment<String>(
                   value: 'Facts & Info',
-                  label: Text(
-                    'Facts & Info',
-                    style: $styles.text.heading3,
-                  ),
+                  label: Text('Facts & Info', style: $styles.text.heading3),
                 ),
                 ButtonSegment<String>(
                   value: 'Trivia',
-                  label: Text(
-                    'Trivia',
-                    style: $styles.text.heading3,
-                  ),
+                  label: Text('Trivia', style: $styles.text.heading3),
```

Comment:
Is there a stylistic reason for not having the trailing comma here and in line 45? I like always having the trailing comma because it makes it readable.

Reply:
Yeah, this is basically just a stylistic preference. If the content fits on one line within a reasonable extent, I prefer to have it be one line rather than spreading this and its short parameters over 4 lines. 
I would have done the same with BorderSide above on l#36, but when I subbed in the `$styles` values it was much longer, so I split it.
I know I have heard all of the indentation and closing parentheses can be fatiguing to readability.

Reply:
Got it! That makes sense. I think Bob had mentioned the potential of having dart fix always adding that trailing comma at some point.